src/kernel_1.jl

```diff
@@ -1,35 +1,18 @@
 function kernel_1(
-  edge_list::kroneckerState{T, R}
-  ) where T<:Integer where R<:Real
+  ij::Array{T,2}
+  ) where T<:Integer
 
+  # Creating Graph
+  ne = maximum(ij)
```

Comment:
`maximum(ij)` is not the number of edges, it's the maximum vertex index. If you assume the vertices are indexed at zero and are consecutive, then `nv(g)` will equal this number plus 1.